### DISCUSSION TITLE
Possible to preserve previous data when manual=true

### DISCUSSION THREAD

**Discussion by andyvanhelsing** [TanStack/query#517] [General] [answered]
Hi there I am new to react-query and I was wondering if there is a built in method to preserve the last data response when a query key changes while in manual mode? Ideally I would like to save the previous results until refetch is called (or use the last data value as initialData). Thanks so much for all your work on this project!

**Comment by cherniavskii** [comment, marked as answer]
See https://github.com/tannerlinsley/react-query/issues/419
